.github/workflows/main.yml

```diff
@@ -536,6 +536,11 @@ jobs:
     - name: Run Python unit tests (Ubuntu ${{ matrix.UbuntuVersion }})
       timeout-minutes: 30
       run: docker exec mlos-build-python-${{ matrix.UbuntuVersion }} make python-test
+    - name: Upload coverage to codecov
```

Comment:
```suggestion
    - name: Upload coverage to codecov
      if: ${{ github.repository == 'microsoft/mlos' }}
```

Reply:
To prevent uploads when the actions are run from a forked repo during testing.

Reply:
Although, would those just go to their own codecov collection?  If so, maybe we don't care and that's useful for testing as well, so nevermind.